seqr/fixtures/variant_searches.json

```diff
@@ -0,0 +1,269 @@
+[
+{
+    "model": "seqr.variantsearch",
+    "pk": 1,
+    "fields": {
+        "guid": "VS0000001_de_novo_dominant_res",
+        "name": "De Novo/ Dominant Restrictive",
+        "search": {
+            "qualityFilter": {
+                "vcf_filter": "pass",
+                "min_ab": 20,
+                "min_gq": 20
+            },
+            "pathogenicity": {
+                "hgmd": [
+                    "disease_causing"
+                ],
+                "clinvar": [
+                    "pathogenic",
+                    "likely_pathogenic"
+                ]
+            },
+            "freqs": {
+                "g1k": {
+                    "ac": null,
+                    "af": 0.001
+                },
+                "gnomad_genomes": {
+                    "ac": null,
+                    "af": 0.001
+                },
+                "gnomad_exomes": {
+                    "ac": null,
+                    "af": 0.001
+                },
+                "exac": {
+                    "ac": null,
+                    "af": 0.001
+                },
+                "topmed": {
+                    "ac": null,
+                    "af": 0.001
+                },
+                "callset": {
+                    "ac": null,
+                    "af": 0.01
+                }
+            },
+            "annotations": {
+                "in_frame": [
+                    "inframe_insertion",
+                    "inframe_deletion"
+                ],
+                "nonsense": [
+                    "stop_gained"
+                ],
+                "frameshift": [
+                    "frameshift_variant"
+                ],
+                "missense": [
+                    "stop_lost",
+                    "initiator_codon_variant",
+                    "start_lost",
+                    "protein_altering_variant",
+                    "missense_variant"
+                ],
+                "extended_splice_site": [
+                    "splice_region_variant"
+                ],
+                "essential_splice_site": [
+                    "splice_donor_variant",
+                    "splice_acceptor_variant"
+                ]
+            },
+            "inheritance": {
+                "filter": {
+                    "A": "has_alt",
+                    "N": "ref_ref"
+                },
+                "mode": "de_novo"
+            }
+        }
+    }
+},
+{
+    "model": "seqr.variantsearch",
+    "pk": 2,
+    "fields": {
+        "guid": "VS0000002_recessive_restrictiv",
+        "name": "Recessive Restrictive",
+        "search": {
+            "qualityFilter": {
```

Comment:
given that the set of filters will evolve, should there be a "search_api_version" field inside "search"?